src/main/resources/misc.yml

```diff
@@ -2,4 +2,17 @@
 Playtime:
   Messages:
     Check-self: "<green>You have been playing for <playtime>."
-    Check-other: "<green><player> has been playing for <playtime>."
\ No newline at end of file
+    Check-other: "<green><player> has been playing for <playtime>."
+
+# Scheduler settings
+Scheduler:
+  Scheduler-tag: '<green>[<red>Scheduler<green>]'
+  Messages:
```

Comment:
In English, the word minute should be in singular form when there is one of something (*1 minute, 2 minutes*).

Perhaps a stop and restart string like `Restart-second` and `Restart-minute`, etc, which is the one broadcasted when there's only one minute or second left.